libraries/plugins/es_objects/es_objects.cpp

```diff
@@ -95,8 +94,16 @@ void es_objects_plugin_impl::updateDatabase( const vector<object_id_type>& ids ,
       limit_documents = _es_objects_bulk_replay;
 
    if (curl && bulk.size() >= limit_documents) { // we are in bulk time, ready to add data to elasticsearech
-      if(!graphene::utilities::SendBulk(curl, bulk, _es_objects_elasticsearch_url, _es_objects_logs, "objects_logs"))
+
+      graphene::utilities::ES es;
+      es.curl = curl;
+      es.bulk_lines = bulk;
+      es.elasticsearch_url = _es_objects_elasticsearch_url;
+      es.auth = _es_objects_auth;
+
+      if(!graphene::utilities::SendBulk(es))
          elog("Error sending data to database");
+         //return false;
```

Comment:
Either remove the commented-out return statement or add curly braces around these two lines